lib/led/rgb.js

```diff
@@ -475,6 +471,55 @@ RGB.prototype.toggle = function() {
   return this[this.isOn ? "off" : "on"]();
 };
 
+/**
+ * pulse Fade the Led in and out in a loop with specified time
+ * @param  {number} duration Time in ms that a fade in/out will elapse
+ * @return {Led}
+ *
+ * - or -
+ *
+ * @param  {Object} val An Animation() segment config object
+ */
+
+RGB.prototype.pulse = function(duration, callback) {
+  var state = priv.get(this);
+
+  var currentColor = state.prev;
+
+  this.stop();
+
+  var options = {
+    duration: typeof duration === "number" ? duration : 1000,
+    keyFrames: [
+      {intensity: 0, color: currentColor},
+      {intensity: 100, color: currentColor}
+    ],
+    metronomic: true,
+    loop: true,
+    easing: "inOutSine",
```

Comment:
Where are the docs for these options? I don't see much at https://github.com/rwaldron/johnny-five/blob/002b5191b99a990bc6e6f34ada62b99d5a01ac09/lib/animation.js#L139-L145

Reply:
See http://johnny-five.io/api/animation/#segment-properties